nginx/etc/nginx/nginx.conf

```diff
@@ -0,0 +1,136 @@
+user nginx;
+worker_processes 4;
+pid /run/nginx.pid;
+
+events {
+    worker_connections 768;
+    # multi_accept on;
+}
+
+http {
```

Comment:
For `http`, I've mostly tried to take the default configuration file provided by the image and only add a few settings:

```nginx
http {
    include /etc/nginx/mime.types;
    default_type application/octet-stream;

    log_format timed_combined '$remote_addr - $remote_user [$time_local] '
                              '"$request" $status $body_bytes_sent $gzip_ratio '
                              '"$http_referer" "$http_user_agent" '
                              '$request_time $upstream_response_time';

    access_log /var/log/nginx/access.log timed_combined;

    sendfile on;

    keepalive_timeout 65;

    server_tokens off;

    gzip on;
    gzip_types application/javascript text/css application/json;
    gzip_disable "msie6";

    set_real_ip_from 10.0.0.0/24;
    set_real_ip_from 10.0.2.0/24;
    set_real_ip_from 10.0.4.0/24;
    set_real_ip_from 10.0.6.0/24;

    real_ip_header X-Forwarded-For;

    include /etc/nginx/conf.d/*.conf;
}
```

Most notably the `timed_combined` configuration, `server_tokens`, `gzip`, and `real_ip_header` stuff.